securedrop/source_templates/base.html

```diff
@@ -26,6 +26,9 @@
       {% endblock %}
 
       <div class="panel selected">
+	{% if 'logged_in' in session %}
+	<a href="{{ url_for('logout') }}" class="btn pull-right">Log Out</a>
+	{% endif %}
```

Comment:
Minor nit: Top right seems like a good location for this (it's a familiar location for a log out button), but I'm not sure if I like it better inside or outside of the `.panel` div. However, that's a minor design think that we'll probably end up revisiting later anyway, so I think it's fine to merge the way it is.